onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -3230,6 +3230,11 @@ void RegisterContribSchemas() {
           "(Optional) SDK version used to convert the model.",
           AttributeProto::STRING,
           OPTIONAL_VALUE)
+      .Attr(
+          "hardware_arch",
```

Comment:
this schema change triggered kernel doc generation failure? seems like something else needs to change? maybe some doc file?

Reply:
Updated the doc as well.